src/Home.js

```diff
@@ -24,16 +24,17 @@ export default class Home extends Component {
             const data = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.props.match.params.pokemon}`)
 
             this.setState({ pokemon: data.body.results })
+            console.log(this.state.pokemon)
         }
     }
-    
+   
     handleSearch = async (e) => {
         e.preventDefault();
         const data = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.state.searchQuery}`)
 
         this.setState({ 
             pokemon: data.body.results, })
-            console.log('data', data)
+           console.log(this.state.pokemon)
 
      //takes the search query and puts it in the url
      this.props.history.push(`/pokemon_list/${this.state.searchQuery}`)
```

Comment:
I think you just want to pass the searchQuery here instead of the whole string. like `this.props.history.push(this.state.searchQuery)`